main/settings.py

```diff
@@ -84,16 +85,18 @@
 
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 
-ACCOUNT_FORMS = {'signup': 'accounts.forms.ExtendedSignupForm'}
-ACCOUNT_USERNAME_REQUIRED = False
+AUTH_USER_MODEL = 'accounts.CustomUser'
+ACCOUNT_SIGNUP_FORM_CLASS = 'accounts.forms.SignupForm'
 ACCOUNT_AUTHENTICATION_METHOD = 'email'
+ACCOUNT_EMAIL_REQUIRED=True
+ACCOUNT_USERNAME_REQUIRED = False
 ACCOUNT_EMAIL_REQUIRED = True
-ACCOUNT_EMAIL_VERIFICATION = 'mandatory'
```

Comment:
Can we leave this in please? I think this is an important step in avoiding getting spammed with signups. Also, in development you should see the verification email in the console and should just be able to copy the link to the browser to confirm the email address which in turn should allow you to log in.
![image](https://user-images.githubusercontent.com/31205982/96988375-280adc00-151c-11eb-80c5-815dd19c216d.png)

Reply:
perfect, hadn't thought of that!